web-api/openapi/front_office.yaml

```diff
@@ -117,6 +117,13 @@ paths:
           required: false
           schema:
             $ref: '#/components/schemas/IncludeProperties'
+        - name: include_snippet
+          in: query
+          description:
+            $ref: '#/components/schemas/IncludeSnippet/description'
+          required: false
+          schema:
+            $ref: '#/components/schemas/IncludeSnippet'
```

Comment:
I wouldn't keep this up-to-date since it is deprecated.

Reply:
Through then it gets out of sync which can make it messy.

As long as it's trivial to do I would prefer to keep it in sync until removed.